src/Twig/FieldExtension.php

```diff
@@ -121,7 +121,9 @@ public function getLabel(Field $field): string
 
     public function getType(Field $field): string
     {
-        return $field->getType() ?? $field->getDefinition()->get('type');
+        // Note, this used to be a ternary, with `?? $field->getDefinition()->get('type');`. I'm not sure
```

Comment:
Hm, looks like it was left as a "safe" choice in #1709 . Which means it's probably safe as-is now.

Reply:
Ok!